std_msgs/Header header
geometry_msgs/Transform origin_transform
float32 OOB_deformability
float32 OOB_sensitivity
float32 OOB_mass
uint8 OOB_r
uint8 OOB_g
uint8 OOB_b
uint8 OOB_a
float64 dvxl_cell_size
geometry_msgs/Vector3 dimensions
uint8[] data
